src/axios.ts

```diff
@@ -0,0 +1,212 @@
+import axios, { AxiosInstance, AxiosPromise, AxiosRequestConfig, AxiosResponse } from 'axios';
+
+import { CreateGraphOptionsSpec, TableMetadata, FileUploadOptionsSpec, EdgesSpec, EdgesOptionsSpec, NodesSpec, OffsetLimitSpec, GraphSpec, RowsSpec, TablesOptionsSpec, UserSpec, WorkspacePermissionsSpec, } from './index';
```

Comment:
Should probably wrap this for readability:
```suggestion
import {
  CreateGraphOptionsSpec,
  TableMetadata,
  FileUploadOptionsSpec,
  EdgesSpec,
  EdgesOptionsSpec,
  NodesSpec,
  OffsetLimitSpec,
  GraphSpec,
  RowsSpec,
  TablesOptionsSpec,
  UserSpec,
  WorkspacePermissionsSpec,
} from './index';
```